core/themes/seven/css/style.css

```diff
@@ -1586,6 +1586,25 @@ body.maintenance-page #logo {
   margin: 0 auto;
 }
 
+/* Links for the simplified login pages. */
+.user-form-links ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+.user-form-links li {
+  display: inline;
+  margin: 0.25em 0;
+}
+.user-form-links li:not(:last-child):after {
+  content: "•";
```

Comment:
So, you remove the dots (li) and then use a dot as separator? :wink: 

Since I read that CSS content's also read aloud by screen readers, I'm no fan of using actual characters as styling element.
We have tons of icons... nothing appropriate available?